.github/workflows/ci.yml

```diff
@@ -81,22 +86,28 @@ jobs:
     - uses: actions/setup-node@v1
       with:
         node-version: ${{ secrets.NODE_VERSION }}
-    - uses: actions/cache@v1
+    - name: Cache node modules
+      id: node-cache
+      uses: actions/cache@v2
       with:
-        path: ~/.npm
-        key: ${{ runner.os }}-node-${{ hashFiles('**/package-lock.json') }}
+        path: |
+          ~/cache
+          ~/.cache
+          !~/cache/exclude
+          **/node_modules
+        key: ${{ runner.os }}-node-${{ hashFiles('**/lockfiles') }}
         restore-keys: |
           ${{ runner.os }}-node-
-    - run: |
+    - name: Install dependencies
+      run: |
         cd ui
-        npm i
-        npm run build && npm run export
+        npm ci
         cd ..
         cd provider-ui
-        npm i
-        npm run build && npm run export
+        npm ci
```

Comment:
This is to check the build and export of Meshery bundle, I don't think npm ci will execute export 🤔 ?

Reply:
Ok, let me get back that to `npm i`. I'm building and exporting Next App [here](https://github.com/ramrodo/meshery/blob/gh-1037-add-e2e-test-for-userpreference/.github/workflows/ci.yml#L188)

Reply:
Got it, then I guess let's get rid of this action @ramrodo . With e2e in place I don't think we need additional UI build check